common-rendering/package.json

```diff
@@ -7,7 +7,7 @@
 	},
 	"dependencies": {
 		"@emotion/react": "^11.4.1",
-		"@guardian/libs": "^3.8.1",
+		"@guardian/libs": "^5.0.0",
```

Comment:
Should this be `^5.1.0` like AR?

Reply:
This is due to a difference between `npm` and `yarn`. I asked for version `^5.0.0.` from both. `npm` set `5.1.0` in both the `package.json` and the lockfile. `yarn` set `5.0.0` in the `package.json` and `5.1.0` in the lockfile.

Either way, it's always `5.1.0` in the lockfile, so that's the version all three are on.

Reply:
Makes perfect sense - thanks